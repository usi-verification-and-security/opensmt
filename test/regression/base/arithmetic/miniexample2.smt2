(set-logic QF_LIA)
(declare-fun x () Int)
(define-fun uninterp_mul ((a Int) (b Int)) Int (* a b))
(assert (= x x))
(assert (= (uninterp_mul 2 x) (+ x x)))
(check-sat)
(get-model)
(exit)