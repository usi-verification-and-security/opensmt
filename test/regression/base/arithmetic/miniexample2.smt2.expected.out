sat
(
  (define-fun x () Int
    0)
)
